src/main/java/wordle/model/WordPoolImpl.java

```diff
@@ -0,0 +1,37 @@
+package wordle.model;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Collections;
+import java.util.List;
+
+public class WordPoolImpl implements WordPool {
+
+	private static final String BASE_DATE = "20210619";
+	private final List<String> words;
+
+	public WordPoolImpl(List<String> words) {
+		this.words = Collections.unmodifiableList(words);
+	}
+
+	@Override
+	public boolean contains(String word) {
+		return words.contains(word);
+	}
+
+	@Override
+	public String getTodayAnswerWord() {
+		return words.get(calculateIndexForTodayAnswerWord());
+	}
+
+	private int calculateIndexForTodayAnswerWord() {
+		return calculatePeriod() % words.size();
+	}
+
+	private int calculatePeriod() {
+		LocalDate now = LocalDate.now();
+		LocalDate past = LocalDate.parse(BASE_DATE, DateTimeFormatter.BASIC_ISO_DATE);
```

Comment:
```suggestion
		private static final String STANDARD_DATE = LocalDate.parse(BASE_DATE, DateTimeFormatter.BASIC_ISO_DATE);
```

`parse`하는 부분도 상수로 사용하면 재사용이 가능해 보입니다. 😊